cmake/functions.cmake

```diff
@@ -157,6 +157,7 @@ function(ml_add_non_distributed_library _target _type)
         "${COVERAGE}")
     elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
       target_link_libraries(${_target} PRIVATE
+        "INCREMENTAL:NO"
```

Comment:
I think the `Release` target adds this automatically and it's probably fine that debug builds don't have it.

Reply:
Without it the linker warns that:
```
LINK : warning LNK4075: ignoring '/INCREMENTAL' due to '/OPT:REF' specification [C:\Users\eds\src\ml-cpp\cmake-build-debug\lib\core\MlCore.vcxproj]
```
So its inclusion was just to quieten the linker warning. .. I'll remove it as its not necessary

Reply:
Oh, I see. Well then the solution is probably not to do `OPT:REF` either for debug builds. It makes sense that debug builds don't do optimisations. Does the syntax you used to only include `/O2` in release builds also work for linker arguments?

Reply:
> Does the syntax you used to only include `/O2` in release builds also work for linker arguments?

Yes, should do, I'll give it a whirl anyway